sre (and dev?) role unable to seal secrets

In sandbox where I am an admin:
```
$ gds sandbox seal -- --token $(gds aws gsp-sandbox -- aws-iam-authenticator token -i sandbox | jq -r '.status.token') <mysecret.json >mysealedsecret.json
$
```

In verify where I am a mere sre:
```
$ gds verify seal -- --token $(gds aws gsp-verify -- aws-iam-authenticator token -i verify | jq -r '.status.token') <mysecret.json >mysealedsecret.json
panic: Error fetching certificate: services "http:gsp-sealed-secrets-controller:" is forbidden: User "alex.monk" cannot get resource "services/proxy" in API group "" in the namespace "gsp-system"
```

--- a/charts/gsp-cluster/templates/00-aws-auth/dev-cluster-role.yaml
+++ b/charts/gsp-cluster/templates/00-aws-auth/dev-cluster-role.yaml
@@ -61,6 +61,10 @@ rules:
     - get
     - list
     - watch
+  - apiGroups: [""]
+    resources: ["services/proxy"]
+    verbs:
+    - get
 
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1

--- a/charts/gsp-cluster/templates/00-aws-auth/sre-cluster-role.yaml
+++ b/charts/gsp-cluster/templates/00-aws-auth/sre-cluster-role.yaml
@@ -61,6 +61,10 @@ rules:
     - get
     - list
     - watch
+  - apiGroups: [""]
+    resources: ["services/proxy"]
+    verbs:
+    - get
 
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1

